pkg/apis/core/validation/validation.go

```diff
@@ -4967,6 +4968,12 @@ func ValidateResourceQuotaResourceName(value string, fldPath *field.Path) field.
 			return append(allErrs, field.Invalid(fldPath, value, isInvalidQuotaResource))
 		}
 	}
+
+	if r := strings.TrimPrefix(value, "count/"); r != value {
+		if len(strings.Split(r, ".")) == 1 && !helper.IsStandardQuotaResourceName(r) {
```

Comment:
helper.IsStandardQuotaResourceName seems to contain incomplete set of resources in core group, e.g. serviceaccount. this means count/serviceaccount would not be allowed with this PR, but it is ligit.

Reply:
@yliaog you're right.

`serviceaccounts ` and `limitranges` are not in  [standardQuotaResources](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/helper/helpers.go#L194). 

```shell
➜  ~ kubectl api-resources --api-group="" --namespaced
NAME                     SHORTNAMES   APIVERSION   NAMESPACED   KIND
bindings                              v1           true         Binding
configmaps               cm           v1           true         ConfigMap
endpoints                ep           v1           true         Endpoints
events                   ev           v1           true         Event
limitranges              limits       v1           true         LimitRange
persistentvolumeclaims   pvc          v1           true         PersistentVolumeClaim
pods                     po           v1           true         Pod
podtemplates                          v1           true         PodTemplate
replicationcontrollers   rc           v1           true         ReplicationController
resourcequotas           quota        v1           true         ResourceQuota
secrets                               v1           true         Secret
serviceaccounts          sa           v1           true         ServiceAccount
services                 svc          v1           true         Service
```

Reply:
I will add `serviceaccounts` and `limitranges ` together to `standardQuotaResources ` in this pr later.

Reply:
what about bindings and endpoints? and events or podtemplates? i think if they are allowed currently for quota, they should be allowed after your PR is merged.

it may be safer by checking "StandardQuotaResourceName + a few additional resources" here.